### DISCUSSION TITLE
Playing Text

### DISCUSSION THREAD

**Discussion by niceEli** [maximmax42/Discord-CustomRP#40] [Ideas]
Can You Make It So You Can Choose What It Says You Are Playing

**Comment by maximmax42** [comment]
Uh, elaborate?
If you mean "Playing **[this text here]**", then that text is the name of the Discord application you create, you can make multiple apps to change that name, it's the only way.

**Comment by niceEli** [comment]
it works now on my machine

**Reply by maximmax42** [reply]
Wow, almost 3 years.
